cli/flox/src/commands/pull.rs

```diff
@@ -593,8 +593,8 @@ mod tests {
 
     /// Pulling an environment without packages for the current platform should
     /// succeed if force is passed
-    #[test]
-    fn test_handle_pull_result_2() {
+    #[tokio::test]
```

Comment:
I dont know if it would be preferable to start a single thread runtime in the test
currently this just calls for confusion becasue the method is async but nothing in it is.
At least add a comment why async/**tokio** is necessary here

Reply:
Added a comment